frontend/pokino-app/src/app/api/json-weather-object.ts

```diff
@@ -8,12 +8,18 @@ export class JsonWeatherObject implements JsonObject {
 
 	constructor() {}
 
-	public static fromJSON(data: any): JsonWeatherObject {
-		const weather: JsonWeatherObject = new JsonWeatherObject();
-		weather.clouds = data.clouds.all;
-		weather.wind = data.wind;
-		weather.location = data.name;
-		return weather;
+	public static fromJSON(data: any): JsonWeatherObject | null {
```

Comment:
das würde ich wohl anpassen und ab jetzt den weather/random endpoint brauchen wie in meinem letzten PR - sonst ist das wetter ein bisschen langweilig. dieser gibt dir ein einfaches weather json zurück und ändert immer die werte - sollte auch im swagger dokumentiert sein